code/mcp_server/cognitive_governance_engine/governance_orchestrator.py

```
@@ -1 +1,130 @@
+        rep_raw = self._extract_protocol_output(session_ctx, "ReasoningAndExplanationProtocol")
+        esl_raw = self._extract_protocol_output(session_ctx, "EmotionalStateLayerProtocol")
+        mtp_raw = self._extract_protocol_output(session_ctx, "MemoryTaggerProtocol")
+
+        if rep_raw and esl_raw and mtp_raw:
+            try:
+                rep_norm = self._adapt_rep_output(rep_raw)
+                coherence = run_coherence_checks(rep_norm, esl_raw, mtp_raw)
+                self.state["coherence"] = coherence
+                if not coherence.get("is_coherent", True):
+                    self.state["warnings"].extend(coherence.get("reasons", []))
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)